src/parser/jobs/sam/modules/Sen.tsx

```diff
@@ -52,14 +53,15 @@ export default class Sen extends Module {
 
 		// Death, as well as all Iaijutsu, remove all available sen
 		this.addHook('cast', {by: 'player', abilityId: IAIJUTSU}, this.remove)
+		this.addHook('cast', {by: 'player', abilityId: TSUBAME}, this.overwrite)
 		this.addHook('death', {to: 'player'}, this.remove)
 
-		// Hagakure because he's a speshul boi
-		this.addHook(
-			'cast',
-			{by: 'player', abilityId: ACTIONS.HAGAKURE.id},
-			this.onHagakure,
-		)
+		// Hagakure because he's was a speshul boi, but now he's dead jim. He's DEAD!
```

Comment:
If he's that dead, you can probably just delete this instead of commenting it out.